nomic/project.py

```diff
@@ -1336,8 +1392,6 @@ def refresh_maps(self):
 
         logger.info(f"Updating maps in project `{self.name}`")
 
-        return self
-
     def get_tags(self) -> Dict[str, str]:
```

Comment:
Type signature is Dict[str, str], but function returns `Dict[str, List[str]]`.

I find it hard to decide if the keys of this dictionary should be the tag names, or the datum ids. Seems like the code here also goes back and forth.

We should think about how to support a workflow that wants to get all the tags associated with an id, not just all the ids associated with a tag. 

TBH, my preferred format would be to return either a list of dicts ([{id, tag}]) or a list of two-tuples ([(id, tag)]).

Reply:
The endpoint from atlas returns a mapping from ids to tags (the first thing you said). I think we should overload this functions parameters to specify if you want id -> tag or tag -> id

But I agree having both this good.

Reply:
Made a card.